app/models/user.rb

```diff
@@ -1,10 +1,17 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable, :confirmable,
+         :recoverable, :rememberable, :validatable
```

Comment:
I encountered this error while trying to sign up as a new user
![image](https://github.com/henask12/Blogify/assets/78251927/92d9bd76-42b4-4691-b526-254f86c67914)

Please fix this to ensure that your app works properly. 😊🛠
- The application should allow for registering a new user.
- A user can log in with a combination of email and password.
- A hashed password is stored in the database.
- The application asks for confirmation of email.
- A user can reset the password in the application